lib/resize.php

```diff
@@ -15,4 +15,23 @@ function resizeImage($image, $max_width, $max_height)
     $new_height = ceil($scale*$old_height);
 
     return imagescale($image, $new_width, $new_height, IMG_TRIANGLE);
-}
\ No newline at end of file
+}
+
+function ResizePngImage($image, $max_width, $max_height)
```

Comment:
Uppercase names indicates usually classes, so this is irritating.

Reply:
Copy paste from print.php, was about to call it from here too there. But sure we can update that.